docs/src/examples/auth_server.rst

```diff
@@ -2,7 +2,7 @@
 
 Authenticating server
 =====================
-The script below starts up a simple Flask web server for authentication.
+The scripts below detail how to start up a simple authentication sever with both Flask and FastAPI.
```

Comment:
Might as well link them here for easy navigation rather than scrolling all the way down. But I can add them later today if you didn't already!